### PR TITLE
Updated symbol names to fit in UI

### PR DESCRIPTION
**Pull request by Adonis2115**
Ran yarn test

### CODE REVIEW

File: tokens/telos.json [hunk 1]
```diff
@@ -364,6 +364,7 @@
     "logoURI": "https://raw.githubusercontent.com/swapsicledex/swapsicle-token-list/master/logos/telos/0x73875789809885DAa0B2334C5CDa5Af6fC3a5454/logo.png",
     "decimals": 18
   },
+
```
Comment (paoloswap) [inline tokens/telos.json]: no empty line please

File: tokens/telos.json [hunk 2]
```diff
@@ -372,9 +373,10 @@
     "decimals": 6,
     "logoURI": "https://lh3.googleusercontent.com/d/1JloVw3WC_5PmZzxqB3MVLD9zvU7Hru3Y"
     },
+
```
Comment (paoloswap) [inline tokens/telos.json]: no empty line please

**Review by paoloswap — CHANGES_REQUESTED** [review]
No empty line please, eventually run a code formatter, because I see also more identation than needed after line 300 or so... maybe was done before. Better fix it ;)

**Review by Adonis2115 — COMMENTED** [review]
Removed Empty lines via vscode regex find & replace

**Review by paoloswap — APPROVED** [review]
Looks good

**Pull request merged**